src/actors/objects/mod.rs

```diff
@@ -9,15 +9,19 @@ use std::time::Duration;
 use bytes::Bytes;
 use failure::{Fail, ResultExt};
 
+#[rustfmt::skip]
 use ::sentry::configure_scope;
+#[rustfmt::skip]
```

Comment:
my rustfmt tried to reformat this to

```
use sentry::...
```

which resolves to the local module instead of the sdk crate